src/ng/services/dom/table.js

```diff
@@ -54,4 +54,16 @@ export default class Table {
 	get data() {
 		return new Data(this.model);
 	}
+
+	focus() {
+		this.markup.table.focus();
+	}
+
+	get target() {
+		return this.markup.table;
```

Comment:
target and document shouldn't be propogated
if you need any event or property for them just make them directly in table.js

Looking something like that (instead of using document directly)
```javascript
table.keyDown.on(e =>
```

Thanks! Kate